test/import-test.js

```diff
@@ -789,7 +793,8 @@ describe('#importDataElement', () => {
     expect(choice.value.constraints[0].path).to.be.empty;
     expect(choice.value.constraints[0].value).to.be.true;
     expect(choice.value.constraints[1]).to.be.instanceof(CodeConstraint);
-    expect(choice.value.constraints[1].path).to.be.empty;
+    expect(choice.value.constraints[1].path).to.have.length(1);
+    expect(choice.value.constraints[1].path).to.eql([id('primitive','concept')]);
```

Comment:
I'm not sure that this test case is correct.  I *thought* that when we put constraints on choice _options_ that the constraint should end up on the individual _option_ itself (rather than on the _choice_ that contains the option).

I don't think we need to look into it for this PR, but could you add a TODO to confirm this test case is correct?

Reply:
okay i'll add a TODO for that.